source/_posts/2021-03-06-android-2021-q1-releases.markdown

```diff
@@ -0,0 +1,95 @@
+---
+title: "Home Assistant Companion Android App 2021 Q1 Releases"
+description: "2021 Q1 Updates for the Home Assistant Companion Android App"
+date: 2021-03-04 00:00:00
+date_formatted: "March 4, 2021"
+comments: true
+author: Daniel Shokouhi
+categories: Release-Notes
+og_image: /images/blog/2021-03-06-android-q1-releases/Companion.png
+---
+
+Hey everyone! It has been quite some time since we last provided an update on all that is new with the Android app. You may have already noticed that we had changed our release versioning to match that of Home Assistant Core. In fact we will be attempting to align our releases more closely to Core releases. This way we can start supporting brand new features quickly, like the recently released [My Home Assistant](https://my.home-assistant.io).
```

Comment:
```suggestion
Hey everyone! It has been quite some time since we last provided an update on all that is new with the Android app. You may have already noticed that we had changed our release versioning to match that of Home Assistant Core. In fact, we will be attempting to align our releases more closely to Core releases. This way we can start supporting brand new features quickly, like the recently released [My Home Assistant](https://my.home-assistant.io).
```